factory/heartbeatV2Components.go

```diff
@@ -142,6 +143,9 @@ func (hcf *heartbeatV2ComponentsFactory) Create() (*heartbeatV2Components, error
 		HardforkTrigger:                             hcf.processComponents.HardforkTrigger(),
 		HardforkTimeBetweenSends:                    time.Second * time.Duration(cfg.HardforkTimeBetweenSendsInSec),
 		HardforkTriggerPubKey:                       hcf.coreComponents.HardforkTriggerPubKey(),
+		KeysHolder:                                  &testscommon.KeysHolderStub{}, // TODO: use the real component once it is completely integrated
```

Comment:
don't use mock / test packages in production code. Please create a `disabled` package somewhere, alongside with a `disabledKeysHolder` and use this instead.

Reply:
Agree, let's pass the real component here. Should be created & stored in crypto components

Reply:
pushed the integration into crypto components